### PR TITLE
React exercise

### PR DESCRIPTION
**Pull request by ismaeljulia-travelperk**

### CODE REVIEW

File: src/utils/RecipeUtils.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export function strToListIngredients(ingredients) {
+    var ingredients_list = []
```
Comment (ghost) [inline src/utils/RecipeUtils.js:2]: We usually go with ES6 `let`

Comment (ghost) [reply src/utils/RecipeUtils.js:2]: In this case you can even use a `const`

File: src/styles/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+import { NavLink as Link } from 'react-router-dom';
+import styled from 'styled-components';
+  
+export const Nav = styled.nav`
```
Comment (ghost) [inline src/styles/NavBar.js:4]: Bice, we also use styled components this should have gave you an idea on how we use them.

File: src/components/RecipeList.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+function RecipeList() {
+    var path = ''
+    const [recipes, setRecipes] = useState(null);
+
+    useEffect(() => {
+        RecipesService.get().then(x => setRecipes(x));
+    }, []);
+
+    console.log(recipes)
```
Comment (ghost) [inline src/components/RecipeList.js:14]: I guess now it is not needed

**Comment by ghost** [comment]
Can you just add a video of the interactions? 
LGTM to me!

**Comment by ismaeljulia-travelperk** [comment]
> Can you just add a video of the interactions? LGTM to me!

done in the Readme! : )

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**